tools/build_travis.py

```diff
@@ -137,6 +137,30 @@
                "rtos" : ["RTOS_1", "RTOS_2", "RTOS_3"],
                "usb"  : ["USB_1", "USB_2" ,"USB_3"],
                }
+     },
+    {"target": "K64F",
+     "toolchains": "GCC_ARM",
+     "tests": {""     : ["MBED_2", "MBED_10", "MBED_11", "MBED_15", "MBED_16", "MBED_17"],
+               "fat"  : ["MBED_A12", "MBED_19", "PERF_1", "PERF_2", "PERF_3"],
```

Comment:
I see the pin defined in MBED_19, why is this failing? I will add the pin defines for KL43Z